x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/execution/search/QlSourceBuilder.java

```diff
@@ -21,6 +22,7 @@
  * the resulting ES document as a field.
  */
 public class QlSourceBuilder {
+    public static final Version FIELDS_API_INTRODUCTION_VERSION = Version.V_7_10_0;
```

Comment:
The name is not very clear - how about `USE_FIELD_API_VERSION` or `FIELD_API_USAGE_VERSION`